src/Views/SystemView/SystemCPUView.vala

```diff
@@ -19,11 +19,11 @@ public class Monitor.SystemCPUView : Monitor.WidgetResource {
         cpu_percentage_label.has_tooltip = true;
         cpu_percentage_label.tooltip_text = (_("Show detailed info"));
 
-        cpu_frequency_label = new LabelRoundy (_("Frequency"));
+        cpu_frequency_label = new LabelRoundy (_("FREQUENCY"));
```

Comment:
I would like to keep lower case with a capital letter. LabelRoundy makes the word uppercase, so no need to write it in uppercase initially. Is it possible?

Reply:
Ah, sure. I just amended my commit to use title-case for these attributes. I also updated all the .po files to be aligned.